server/graphql/v2/object/Order.js

```diff
@@ -112,6 +112,17 @@ export const Order = new GraphQLObjectType({
           return models.PaymentMethod.findByPk(order.PaymentMethodId);
         },
       },
+      platformFee: {
```

Comment:
It's time for us to find a more positive way to express this than `platformFee`. Exposing this field in the API is the right moment to think about it.

`onTopAmount` ? Any other idea @kewitz and @sbinlondon ?

Reply:
I think that works. Perhaps it would be clearer to move towards something like `platformContributionAmount` since these are not properly fees and neither can be called donations.

Reply:
Agreed to a separate PR for consistency